src/sass/print.scss

```diff
@@ -256,11 +256,12 @@
 
   /* Set selected checkboxes with thick border */
   .block label.checked {
-    border: 3px solid black;
+    border: 1px solid transparent;
     background-color: #fff;
     color: #000;
     font-weight: 600;
     opacity: 1;
+    outline: 3px solid #000;
```

Comment:
Switched to outline instead of border. Border restricts the inside space and outline does not 👍